src/libs/FeedUtils.ts

```diff
@@ -0,0 +1,344 @@
+import type {LocaleContextProps} from '@components/LocaleContextProvider';
```

Comment:
I think we should rename this file to `CardFeedUtils` because most devs might not understand what is a "feed". When it has "card" in the name - the meaning should be more obvious